scripts/test.sh

```diff
@@ -2,16 +2,33 @@
 
 echo "Starting Flame Tester"
 echo "---------------------"
+root_dir=$(pwd)
 for file in $(find . -type d -name "test"); do
   dir=$(dirname $file)
   cd $dir
   echo "Testing $dir"
-  flutter test
+  flutter test --coverage
   test_result=$?
   if [ $test_result -ne 0 ]; then
     exit $test_result
   fi
+
+  if [ -f ".min_coverage" ]; then
+    min_coverage=$(cat .min_coverage)
+
+    coverage_summary=$(lcov --summary coverage/lcov.info)
+    echo "$coverage_summary"
+
+    current_cov=$(echo "$coverage_summary" | grep lines | grep -Eow '\d+\.\d+')
```

Comment:
```suggestion
    current_cov=$(echo "$coverage_summary" | grep lines | grep -Pow '\d+\.\d+')
```

Reply:
damn, with this change, the script works on linux, but not on Mac :(, I think I will not use grep and come with more cross platform solution.